src/main/java/com/washingtonpost/arc/ans/v0_3/model/ANS.java

```diff
@@ -22,7 +22,7 @@
     @Type(value = Media.class, name = "media"),
     @Type(value = Image.class, name = "image"),
     @Type(value = Video.class, name = "video"),
-    @Type(value = Story.class, name = "story"),
+    @Type(value = Collection.class, name = "story"),
```

Comment:
I don't think this is right.  We want to instantiate an actual Story.class when a JSON block declares itself of type "story"

Likewise, you should add a new Type annotation that maps "Gallery.class" to the name="gallery"